vars/parseStageInfo.groovy

```diff
@@ -112,5 +112,31 @@ def call(Map config = [:]) {
     if (config['ftest_arg']) {
       result['ftest_arg'] = config['ftest_arg']
     }
+
+    if (env.STAGE_NAME.contains('run_test.sh')) {
```

Comment:
run_test.sh name is going to go away, possibly soon.

Since there is no information to parse from the 'run_test.sh" as to what the stage is, the defaults for it should be the defaults for the "unitTest.groovy" and "unitTestPost.groovy" and not specified here.

Reply:
I fixed this, now I'm modifing the appropriate defaults from daos Jenkinsfile. In this parseStageInfo section I still have the set up of with_valgrind variable, though.